package.json

```diff
@@ -59,5 +59,8 @@
     "rollup-plugin-copy": "^3.3.0",
     "sass": "^1.45.1",
     "w3c-xmlhttprequest": "^3.0.0"
+  },
+  "dependencies": {
```

Comment:
I don't know why this was added; but please remove.  No changes should be made to the `package.json` that could cause problems; should be no need for it in the `igv.js` code